migramongo-core/src/main/java/com/rinoto/migramongo/MigraMongo.java

```diff
@@ -354,6 +354,7 @@ private MigrationEntry executeMigrationScript(MongoMigrationScript migrationScri
             migrationScript.migrate(database);
             return migrationHistoryService.setMigrationStatusToFinished(migrationEntry);
         } catch (Exception e) {
+            logger.error("Migration Script failed with Excetion", e);
```

Comment:
I thought we already had this one :)

Could you:
- fix the typo (Excetion)
- include the migratonEntry info in the log (fromVersion toVersion)